admin/aws.py

```diff
@@ -397,6 +401,11 @@ class FakeAWS(object):
     """
     def __init__(self):
         self.cloudfront_invalidations = []
+        for bucket in self.s3_buckets.values():
+            for key, contents in bucket.items():
+                if isinstance(contents, basestring):
```

Comment:
It might also make sense to add an alternate constructor which requires the PRecord thing rather than overloading `__init__` like this.